src/sentry/rules/history/preview.py

```diff
@@ -278,46 +311,49 @@ def get_events(
         if dataset.value == Dataset.Transactions.value:
             # this query cannot be made until https://getsentry.atlassian.net/browse/SNS-1891 is fixed
             """
+            kwargs["having"] = kwargs["conditions"]
```

Comment:
need to move this line too?

Reply:
I just wanted them to be in the same order

Reply:
ah ok. i just saw the one line moved out of the comments and wasn't sure if we needed to move the other